src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationExternalController.java

```diff
@@ -228,6 +231,56 @@
 
     }
 
+    @ApiOperation(
+            value = "Retrieves all Organisations of requested status for user"
+                    + " with minimal e.g. organisationIdentifier, name and contact information if address flag is true",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "Successfully retrieved list of all Organisations of"
+                            + " requested status with minimal information",
+                    response = OrganisationMinimalInfoResponse.class,
+                    responseContainer = "list"
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = "Forbidden Error: Access denied for either invalid permissions or user is pending"
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = "No Organisation found"
+            ),
+            @ApiResponse(
+                    code = 401,
+                    message = "Unauthorized Error : The requested resource is restricted and requires authentication"
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = "Internal Server Error"
+            )
+    })
+    @GetMapping(
+            path = "/status/{status}",
+            produces = APPLICATION_JSON_VALUE
+    )
+    @Secured({"pui-organisation-manager", "pui-finance-manager", "pui-case-manager", "pui-caa", "pui-user-manager"})
+    public ResponseEntity<List<OrganisationMinimalInfoResponse>>
+        retrieveOrganisationsByStatusWithAddressDetailsOptional(
+            @ApiParam(hidden = true) @UserId String userId,
+            @PathVariable("status") String status,
+            @ApiParam(name = "address")
+            @RequestParam(value = "address", required = false, defaultValue = "false") boolean address) {
```

Comment:
remove request param and use @ApiParam two are not required here.

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/694/commits/d9b19d3fcf78520dfb43772e437a7beeeb07bcb7